cluster-configuration-example/kubernetes.yaml

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+kubernetes:
+  # Find the namesever in  /etc/resolv.conf
+  cluster-dns: IP
+  # To support k8s ha, you should set an lb address here.
+  # If deploy k8s with single master node, please set master IP address here
+  load-balance-ip: IP
+  # specify an IP range not in the same network segment with the host machine.
+  service-cluster-ip-range: A.B.C.D/X
+  # According to the etcdversion, you should fill a corresponding backend name.
+  # If you are not familiar with etcd, please don't change it.
+  storage-backend: etcd2
```

Comment:
>etcd2 [](start = 19, length = 5)

should be etcd3 now

Reply:
The upgrade Pr is blocking. Util now, it's 2.